hack/tools/go.mod

```diff
@@ -80,7 +79,6 @@ require (
 	github.com/google/uuid v1.3.0 // indirect
 	github.com/googleapis/enterprise-certificate-proxy v0.2.5 // indirect
 	github.com/googleapis/gax-go/v2 v2.12.0 // indirect
-	github.com/hashicorp/errwrap v1.1.0 // indirect
 	github.com/hashicorp/hcl v1.0.0 // indirect
```

Comment:
This indirect dep is brought in by `viper` 

```
> go mod why -m github.com/hashicorp/hcl

# github.com/hashicorp/hcl
sigs.k8s.io/cluster-api/cmd/clusterctl/client/config
github.com/spf13/viper
github.com/spf13/viper/internal/encoding/hcl
github.com/hashicorp/hcl
```

We have a number of other indirect hashicorp dependencies in our `go.sum` - but I think all of them are going to remain at MPL for now.